src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -22,31 +22,38 @@
 
 //	Bookmarks of places to mine. Add additional bookmarks separated by comma.
 string[] SetMiningSiteBookmark = new[] {
-	"mining_site_bookmark_name",
+	"Asteroid Belts",
 	};
 
 //	Bookmark of location where ore should be unloaded.
-string UnloadBookmark = "station_or_POS_bookmark_name";
+string UnloadBookmark = "home";//home is a custom save location of station in my system who called home 
```

Comment:
for use this setting you need to create a saved location , name it home , is where the bot need to back for sell the farm